discover/portal/articles/03-collaboration/01-managing-documents-and-media/06-using-liferay-sync-desktop.markdown

```diff
@@ -65,16 +65,27 @@ Next, you'll configure the Sync client.
 Now that you've installed Sync, you're ready to configure it! The configuration 
 steps for Sync on Windows and Mac are identical. 
 
-1. Open Sync and enter your Liferay instance's address along with your account 
-   credentials. Click *Sign In* when you're finished. 
+1.  Open Sync and enter your @product@ instance's address along with your 
+    account credentials. Click *Sign In* when you're finished. 
 
-    ![Figure 2: The first time you run Liferay Sync, you need to tell it how to communicate with your Liferay server.](../../../images/sync-setup-01.png)
+    ![Figure 2: The first time you run Liferay Sync, you need to tell it how to communicate with your @product@ server.](../../../images/sync-setup-01.png)
 
-2. Select the sites you want to sync with. You can search for a site in the 
-   *Search* bar above the site list. If you want to sync all the subfolders of 
-   your selected sites, click *Proceed* and move on to the next step. 
+    When connecting to a server via HTTPS, an error appears if the certificate 
+    can't be verified. Choosing *Proceed Anyway* bypasses verification and 
+    leaves the connection open to compromise. Liferay Sync attempts to read the 
+    certificates specified in the Java Control Panel 
+    ([see section 20.4.5](https://docs.oracle.com/javase/8/docs/technotes/guides/deploy/jcp.html#A1152831)). 
+    If Java isn't installed, you can also put your certificates in 
+    [user.home]/.liferay-sync-3/certificates. Liferay Sync will trust all 
+    certificates. 
```

Comment:
Is this sentence correct? "Liferay Sync will trust all certificates"?

Should it be: "Liferay Sync will trust all certificates in this folder"?